Increase fuziness on recently imported WPT flexbox and grid tests.

sammygill opened the pull request:
#### a66a1f29bb367d939c3b4e7f03882e46597a0855
<pre>
Increase fuziness on recently imported WPT flexbox and grid tests.
<a href="https://bugs.webkit.org/show_bug.cgi?id=246576">https://bugs.webkit.org/show_bug.cgi?id=246576</a>
rdar://101207880

Reviewed by Tim Nguyen.

These tests started to fail locally on the iOS simulator after a recent
WPT import. When looking at the actual results, however, they seemed
fine. They were also passing without any issues on EWS. It seems fine to
increase the fuziness on these tests.

* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-001.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-002.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-003.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-004.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-005.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-001.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-002.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-003.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-004.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/layout-algorithm/flex-sizing-rows-indefinite-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/layout-algorithm/grid-template-flexible-rerun-track-sizing.html:

Canonical link: <a href="https://commits.webkit.org/255658@main">https://commits.webkit.org/255658@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/cbff483dc59f7ec7cd405dc22ba3f385838f7dc6

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/6/builds/93046 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/77/builds/2248 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/43/builds/23631 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/8/builds/102753 "Built successfully") | [✅ 🛠 🧪 win](https://ews-build.webkit.org/#/builders/10/builds/162979 "Built successfully and passed tests") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/76/builds/2258 "Built successfully") | [✅ 🛠 mac-debug](https://ews-build.webkit.org/#/builders/71/builds/30575 "Built successfully") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/36/builds/85428 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/12/builds/98871 "Built successfully") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/19/builds/98712 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/78/builds/1552 "Passed tests") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/61/builds/79520 "Built successfully") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/35/builds/28467 "Passed tests") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/9/builds/83466 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/3/builds/83192 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/34/builds/71579 "Passed tests") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/66/builds/36969 "Built successfully") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/73/builds/17092 "Passed tests") | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/67/builds/34782 "Built successfully") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/70/builds/18298 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/74/builds/3920 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/63/builds/38654 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/60/builds/40887 "Passed tests") | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/65/builds/40580 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/62/builds/37489 "Passed tests") | | 
<!--EWS-Status-Bubble-End-->

nt1m commented on LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-001.htm:
```diff
@@ -7,7 +7,7 @@
         <link rel="match" href="reference/align-content-001-ref.html" />
         <meta name="assert" content="This test checks that a multi-line flex container with 'align-content: center' centers lines in the flex container." />
-        <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-5709" />
+        <meta name="fuzzy" content="maxDifference=1; totalPixels=6217" />
```
If you don't use `0-*` ranges, it means you expect _exactly_ a certain maxDifference/totalPixels, and any other number (even lower), will cause the test to fail.

This is not great in case we progress, but also not accurate since other browsers/platforms may have different fuzziness.

```suggestion
        <meta name="fuzzy" content="maxDifference=0-1; totalPixels=0-6217" />
```

sammygill replied:
I see, thanks Tim! I think it should be better now. Let me know what you think.

nt1m reviewed (changes requested):
Please use `0-*` ranges for correctness.

nt1m reviewed (commented):
Please make sure to export to WPT, this command should work assuming HEAD represents this commit.

`export-w3c-test-changes -g HEAD -b 246576 -c --no-linter`

sammygill committed:
Increase fuziness on recently imported WPT flexbox and grid tests.
https://bugs.webkit.org/show_bug.cgi?id=246576
rdar://101207880

Reviewed by Tim Nguyen.

These tests started to fail locally on the iOS simulator after a recent
WPT import. When looking at the actual results, however, they seemed
fine. They were also passing without any issues on EWS. It seems fine to
increase the fuziness on these tests.

* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-001.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-002.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-003.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-004.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-content-005.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-001.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-002.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-003.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-flexbox/align-items-004.htm:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/layout-algorithm/flex-sizing-rows-indefinite-height.html:
* LayoutTests/imported/w3c/web-platform-tests/css/css-grid/layout-algorithm/grid-template-flexible-rerun-track-sizing.html:

Canonical link: https://commits.webkit.org/255658@main
